### PR TITLE
frio: Revert guid truncation for scrollToItem() and improve dropItem()

### PR DESCRIPTION
**Pull request by Hypolite**
Fixes #3208 and #3194

This is a fix of a fix as well as a cleanup PR.

Both `scrollToItem()` and `dropItem()` get a revamp by following the same pattern of getting the relevant DOM element with `document.getElementById()`. this allows Hubzilla guid to be used as is in `id` attributes.

Beyond that, this PR adds missing item dropping support to search result, although it’s been long enough without anyone complaining that I doubt anyone is using this feature.

### CODE REVIEW

File: view/theme/frio/js/textedit.js [hunk 1]
```diff
@@ -162,21 +162,29 @@ function qCommentInsert(obj,id) {
...
-			$.get(url).done(function() {
-				$(document.getElementById(object)).remove();
+
+		var $el = $(document.getElementById(elementId));
+
+		$el.fadeTo('fast', 0.33, function () {
+			$.get(url).then(function() {
+				$el.remove();
+			}).error(function() {
+				// @todo Show related error message
```
Comment (rabuzarus) [inline view/theme/frio/js/textedit.js]: please use the doxygen style `/// @todo`   (3 slashes)

Comment (annando) [inline view/theme/frio/js/textedit.js]: @rabuzarus do you consider this documentation thing as blocker before accepting it?

Comment (rabuzarus) [inline view/theme/frio/js/textedit.js]: No, it is OK :-)

**Comment by rabuzarus** [comment]
@Hypolite 
there seems to be a bug in `dropItem()`. JS console says `Uncaught TypeError: $.get(...).then(...).error is not a function(…)`

**Comment by Hypolite** [comment]
Noted.

**Pull request merged**